kubernetes-model-generator/generateModelDocker.sh

```diff
@@ -20,7 +20,7 @@ set -e
 BASEDIR=$(dirname "$BASH_SOURCE")
 ABSOLUTE_BASEDIR=$(realpath "$BASEDIR/..")
 
-DOCKER_IMAGE_GOLANG=marcnuri/golang-1.20-java17
+DOCKER_IMAGE_GOLANG=marcnuri/golang-1.21-java17
```

Comment:
@rohanKanojia This should be updated for the rest of the models

Reply:
Umm, sorry I don't understand.

Reply:
:facepalm: for some reason I read that this was in a knative subdirectory.
Please close the issue I created indicating that this is a global value.

Reply:
:+1: